### PR TITLE
Feat: assign new admin

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- assign new admin - moderator app

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/moderation/src/components/app-routes.tsx [hunk 1]
```diff
@@ -48,7 +50,8 @@ import routes, {
 const AppRoutes: React.FC<RootComponentProps> = props => {
   const loginQuery = useGetLogin();
```
Comment (quininez) [inline ui/apps/moderation/src/components/app-routes.tsx]: we no longer use this hook, replace it with useGetMyProfileQuery from @akashaorg/ui-awf-hooks/lib/generated/hooks-new

File: ui/apps/moderation/src/extensions/moderate-modal.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ const ModerateModalComponent = (props: RootExtensionProps) => {
       footerUrl1={`${window.location.protocol}//${window.location.host}/@akashaorg/app-legal/code-of-conduct`}
       cancelLabel={t('Cancel')}
       errorText={moderateMutation.error ? `${moderateMutation.error}` : ''}
-      user={loginQuery.data?.pubKey || ''}
+      user={loginQuery.data?.id || ''}
```
Comment (quininez) [inline ui/apps/moderation/src/extensions/moderate-modal.tsx:71]: also here, loginQuery should be replaced with loggedProfileData coming from the new generated hook, look in app-akasha-integration/app-routes for an example

File: ui/apps/moderation/src/extensions/report-modal.tsx [hunk 3]
```diff
@@ -111,7 +111,7 @@ const ReportModalComponent = (props: RootExtensionProps) => {
       cancelLabel={t('Cancel')}
       reportLabel={t('Report')}
       errorText={reportMutation.error ? `${reportMutation.error}` : ''}
-      user={loginQuery.data?.pubKey || ''}
+      user={loginQuery.data?.id || ''}
```
Comment (quininez) [inline ui/apps/moderation/src/extensions/report-modal.tsx:114]: also here

**Review by quininez — CHANGES_REQUESTED** [review]

File: ui/apps/moderation/src/extensions/moderate-modal.tsx (hunk 2 above)
Comment (josenriagu) [reply ui/apps/moderation/src/extensions/moderate-modal.tsx:71]: The modal extension files are deprecated. I will remove them in the second PR

File: ui/apps/moderation/src/components/app-routes.tsx [hunk 4]
```diff
@@ -48,9 +50,15 @@ import routes, {
...
+  const profileDataReq = useGetMyProfileQuery(null, {
+    select: resp => {
+      return resp.viewer?.profile;
+    },
+  });
+
+  const loggedProfileData = profileDataReq.data;
 
-  const checkModeratorQuery = useCheckModerator(loginQuery.data?.id);
+  const checkModeratorQuery = useCheckModerator(loggedProfileData?.id);
```
Comment (quininez) [inline ui/apps/moderation/src/components/app-routes.tsx]: you should use loggedProfileData.did.id, id is an internal database reference, whereas the did.id replaces the pubKey, that is why I pointed you to the example in app-akasha-integration

**Review by quininez — CHANGES_REQUESTED** [review]

**Review by quininez — APPROVED** [review]

**Pull request merged**